client/src/test-runner/runner.ts

```diff
@@ -377,13 +292,13 @@ export class ElmTestRunner {
         if (!this.currentSuite) {
           throw new Error("not loading?");
         }
-        const event: EventTestCompleted = {
+        const event: TestCompleted = {
           ...result.event,
           messages: this.popMessages(),
         };
-        const labels: string[] = [...event.labels];
-        const id = this.addEvent(this.currentSuite, labels, event);
-        this.eventById.set(id, event);
+        // const labels: string[] = [...event.labels];
```

Comment:
Should these commented out lines be removed then?

Reply:
ups. done.